backend/manager/modules/bll/src/main/java/org/ovirt/engine/core/bll/storage/disk/image/BaseImagesCommand.java

```diff
@@ -384,6 +384,9 @@ protected void endSuccessfully() {
                                 newStorageDomainID,
                                 getDestinationDiskImage());
                     }
+
+                    // Update disk's size if it's changed
```

Comment:
in engine's terminology it's 'image', no?

Reply:
yes, but the image with the updated size doesn't exist yet because we're creating the image with the updated size, this essentially update's the disk's size, but I'm not too attached to this phrasing